libraries/consul_installation_binary.rb

```diff
@@ -86,6 +86,7 @@ def self.binary_basename(node, resource)
         case node['kernel']['machine']
         when 'x86_64', 'amd64' then ['consul', resource.version, node['os'], 'amd64'].join('_')
         when /i\d86/ then ['consul', resource.version, node['os'], '386'].join('_')
+        when /^arm/ then ['consul', resource.version, node['os'], 'arm'].join('_')
```

Comment:
Sorry but I think I don’t get your question? Maybe I should elaborate on what my problem was.

I was trying to use this cookbook to install consul on a raspberry pi. Doing so threw an error “403 Forbidden” since the package url was `…/consul_0.7.0_linux_armv7l.zip`. This is not the official downloadable package path. Its simply `…/consul_0.7.0_linux_arm.zip`.

Also it has to be a regex since there are different arm versions but they all start with `arm…`.

Does that answer your question? I am also happy to change if I got something wrong here.